src/Kobo/System.cpp

```diff
@@ -426,3 +429,108 @@ KoboSetBacklightBrightness([[maybe_unused]] int percent)
   }
 #endif
 }
+
+/**
+ * Gets the file that contains the background colour
+ * as text of integer 0-10
+ *
+ * Some models have more than one possibility and each
+ * possibility must be checked in a certain order
+ *
+ * @return the file that contains the background colour
+ * code or nullptr if background colour not supported
+ */
+const char *
+KoboGetBacklightColourFile() noexcept
+{
+#ifdef KOBO
+  const char * colour_files[3] = {
```

Comment:
This array should be `const` (or better: `constexpr`)

Reply:
Yup.  Squeeze all you can into compile!  Changed to constexpr const char * colour_files[3] = {

Reply:
It was initialized at compile time already, but the difference is that it can now be in the `.rodata` section where it will be write-protected.